lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.spec.ts

```diff
@@ -879,6 +1102,7 @@ describe('ContentMetadataComponent', () => {
         });
     });
 
+    // eslint-disable-next-line ban/ban
```

Comment:
no need for this comment (no need to disable eslint) because you added it earlier because here was xdescribe but here is no xdescribe anymore so comment can be removed

Reply:
Done